src/interpreter/ByteCode.h

```diff
@@ -2282,14 +2282,14 @@ class ThrowOperation : public ByteCode {
 
 class ThrowStaticErrorOperation : public ByteCode {
 public:
-    ThrowStaticErrorOperation(const ByteCodeLOC& loc, char errorKind, const char* errorMessage, AtomicString templateDataString = AtomicString())
+    ThrowStaticErrorOperation(const ByteCodeLOC& loc, uint8_t errorKind, const char* errorMessage, AtomicString templateDataString = AtomicString())
         : ByteCode(Opcode::ThrowStaticErrorOperationOpcode, loc)
         , m_errorKind(errorKind)
         , m_errorMessage(errorMessage)
         , m_templateDataString(templateDataString)
     {
     }
-    char m_errorKind;
+    uint8_t m_errorKind;
```

Comment:
Why don't you use `ErrorCode`?

Reply:
I just want to clarify the type and size of member, that's all.
(It's not clear but some compiler may allocate different size for each enum class type)